pydomcfg/tests/bathymetry.py

```diff
@@ -44,6 +45,60 @@ def flat(self, depth: float) -> Dataset:
         ds["Bathymetry"] = xr.full_like(ds["glamt"], depth)
         return _add_attributes(_add_mask(ds))
 
+    def channel(self, depth: float, stiff=1.0) -> Dataset:
+        """
+        Channel with seamount case.
+
+        Produces bathymetry of a channel with a Gaussian seamount in order to
+        simulate an idealised test case. Based on Marsaleix et al., 2009
+        doi:10.1016/j.ocemod.2009.06.011 Eq. 15.
+
+        Parameters
+        ----------
+        depth: float
+            Bottom depth (units: m).
+        stiff: float
+            Scale factor for steepness of seamount (units: None)
+
+        Returns
+        -------
+        Dataset
+        """
+        ds = self._coords
+
+        # Get horizontal grid
+        glamt, gphit = ds.variables["glamt"], ds.variables["gphit"]
+
+        # Find half way point for sea mount location
+        jpi, jpj = ds.dims["x"] // 2, ds.dims["y"] // 2
+        glamt_mid, gphit_mid = glamt[jpj, jpi], gphit[jpj, jpi]
```

Comment:
Don't assume we know the dimension order (i.e., xy or yx), that might change...
You have a few options...
E.g.,
```python
glamt_mid,  gphit_mid = (g.isel(x=jpi, y=jpj) for g in (glamt, gphit))
```